### PR TITLE
Pluginsd doc

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Update documentation for `plugins.d` adding missing information for chart label.

##### Test Plan
PR is only changing documentation so there is not any necessary test.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? Documentation
- Can they see the change or is it an under the hood? If they can see it, where? When users access dashboard, they will have additional information related to fields that can be used to define their chars
- How is the user impacted by the change? With this additional knowledge users can define their charts better.
- What are there any benefits of the change? A correct documentation.
</details>

### CODE REVIEW

File: collectors/plugins.d/README.md [hunk 1]
```diff
@@ -320,6 +322,26 @@ The `value` is floating point (Netdata used `long double`).
...
 
+#### CLABEL
+
+> CLABEL name value source
+
+`CLABEL` defines a label used to organize and identify a chart.
+
+Label names should use alphanumeric characters.
+
+Value is also a alphanumeric field and when more than one word is needed, it is necessary to use quotes.
```
[the remark this answers is not in the document]
Comment (ilyam8) [reply collectors/plugins.d/README.md:333]: @thiagoftsm 


```md
| Character           | Symbol | Label Name | Label Value |
|---------------------|:------:|:----------:|:-----------:|
| UTF-8 character     | UTF-8  |     _      |    keep     |
| Lower case letter   | [a-z]  |    keep    |    keep     |
| Upper case letter   | [A-Z]  |    keep    |    [a-z]    |
| Digit               | [0-9]  |    keep    |    keep     |
| Underscore          |   _    |    keep    |    keep     |
| Minus               |   -    |    keep    |    keep     |
| Plus                |   +    |     _      |    keep     |
| Colon               |   :    |     _      |    keep     |
| Semicolon           |   ;    |     _      |      :      |
| Equal               |   =    |     _      |      :      |
| Period              |   .    |    keep    |    keep     |
| Comma               |   ,    |     .      |      .      |
| Slash               |   /    |    keep    |    keep     |
| Backslash           |   \    |     /      |      /      |
| At                  |   @    |     _      |    keep     |
| Space               |  ' '   |     _      |    keep     |
| Opening parenthesis |   (    |     _      |    keep     |
| Closing parenthesis |   )    |     _      |    keep     |
| Anything else       |        |     _      |      _      |
```

[the remark this answers is not in the document]
Comment (thiagoftsm) [reply collectors/plugins.d/README.md]: We thought the same thing, but you were quicker than me, thanks! :laughing:

File: collectors/plugins.d/README.md [hunk 2]
```diff
@@ -116,15 +116,17 @@ For example, if your plugin wants to monitor `squid`, you can search for it on p
...
+Netdata parses 9 lines starting with:
+
+-    `CHART` - create or update a chart
+-    `DIMENSION` - add or update a dimension to the chart just created
+-    `BEGIN` - initialize data collection for a chart
+-    `SET` - set the value of a dimension for the initialized chart
+-    `END` - complete data collection for the initialized chart
+-    `FLUSH` - ignore the last collected values
+-    `DISABLE` - disable this plugin
+-    `CLABEL` - Define label for a specific chart
```
Comment (ilyam8) [inline collectors/plugins.d/README.md:128]:
```suggestion
-    `CLABEL` - add a label to a chart
```

File: collectors/plugins.d/README.md [hunk 3]
```diff
@@ -116,15 +116,17 @@ For example, if your plugin wants to monitor `squid`, you can search for it on p
...
+-    `CLABEL_COMMIT` - merge label definition for a chart.
```
Comment (ilyam8) [inline collectors/plugins.d/README.md:129]:
```suggestion
-    `CLABEL_COMMIT` - commit added labels to the chart.
```

File: collectors/plugins.d/README.md [hunk 4]
```diff
@@ -320,6 +322,46 @@ The `value` is floating point (Netdata used `long double`).
 
 Variables are transferred to upstream Netdata servers (streaming and database replication).
 
+#### CLABEL
+
+> CLABEL name value source
+
+`CLABEL` defines a label used to organize and identify a chart.
+
+Name and value accept characters according the following table:
```
Comment (ilyam8) [inline collectors/plugins.d/README.md:331]:
```suggestion
Name and value accept characters according to the following table:
```

File: collectors/plugins.d/README.md [hunk 5]
```diff
@@ -320,6 +322,46 @@ The `value` is floating point (Netdata used `long double`).
...
+| At                  |   @    |     _      |    keep     |
+| Space               |  ' '   |     _      |    keep     |
+| Opening parenthesis |   (    |     _      |    keep     |
+| Closing parenthesis |   )    |     _      |    keep     |
+| Anything else       |        |     _      |      _      |
+
+The `source` is an integer field that can have the following values:
+- `1`: The value was set automatically
+- `2`: The value was set manually
+- `4`: This is a K8 label
```
Comment (ilyam8) [inline collectors/plugins.d/README.md:358]:
```suggestion
- `1`: The value was set automatically.
- `2`: The value was set manually.
- `4`: This is a K8 label.
```

File: collectors/plugins.d/README.md [hunk 6]
```diff
@@ -320,6 +322,46 @@ The `value` is floating point (Netdata used `long double`).
...
+
+The `source` is an integer field that can have the following values:
+- `1`: The value was set automatically.
+- `2`: The value was set manually.
+- `4`: This is a K8 label.
+- `8`: This is a label defined using `netdata` agent cloud link.
+
+#### CLABEL_COMMIT
+
+`CLABEL_COMMIT` will indicate to netdata that all labels were defined and chart can be updated.
```
Comment (ilyam8) [inline collectors/plugins.d/README.md:363]:
```suggestion
`CLABEL_COMMIT` indicates that all labels were defined and the chart can be updated.
```

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Review by DShreve2 — APPROVED** [review]

**Pull request merged**